etc/korrel8r/korrel8r.yaml

```diff
@@ -8,6 +8,8 @@ stores:
     lokiStack: 'https://{{(get "k8s:Route:{namespace: openshift-logging, name: logging-loki}" | first).Spec.Host}}'
   - domain: metric
     metric: 'https://{{(get "k8s:Route:{namespace: openshift-monitoring, name: thanos-querier}" | first).Spec.Host}}'
+  - domain: netflow
+    lokiStack: 'https://{{(get "k8s:Route:{namespace: netobserv, name: loki}" | first).Spec.Host}}'
```

Comment:
is this something the end user will be able to configure? This route for loki is the default one indeed, but users can install LokiStack with a different name and namespace

Reply:
Yes, this is the default config file for korrel8r, which assumes standard openshift locations, it can be edited.